### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by gabyse1**
# Changes to the project 😃  🆕 
- Create a header partial view to display a New Post button and add it to corresponding main views.
- Create the New Post view to create new posts for a given user.
- Create the Post controller actions new and create to insert new posts to the database.
- Create the New Comment view to add new comments comments for a given user.
- Create the new and create Comments Controller actions to insert new comments to the database.
- Update the Like model to allow the user make only one like for a given post.
- Create the create and destroy Likes Controller actions to allow user like and unlikes posts. 
- Add comments and lines routes to the routes.rb file.
- Refactor the Comments and likes box
- Add some styles to the New Post and New Comment forms.
- Update the README.md file.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
+    resources :posts, only: %i[index new create show] do
+    end
```
Comment (descholar-ceo) [inline config/routes.rb:5]: We should only embed our resources as follow: 
-> comments inside a post route, 
-> likes inside a post

Try something like this:
```suggestion
     resources :comments, only: %i[index new create show]     
     resources :likes, only: %i[create destroy]
 end
```

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @gabyse1 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (gabyse1) [reply config/routes.rb:5]: ## Hi @descholar-ceo 😃 

#### First of all thank you very much for the review. 👍 

I am going to nest the routes as you suggested me.

Kindly I would like to ask you, There will be no difficulty if I nest the routes more than one level deep?
I read a little about it at https://guides.rubyonrails.org/routing.html#nested-resources.

Thank you for your time.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @gabyse1, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: config/routes.rb [hunk 2]
```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
+    resources :posts, only: %i[index new create show] do
+      resources :comments, only: %i[new create]
+      resources :likes, only: %i[create destroy]
+    end
```
[the remark this answers is not in the document]
Comment (gabyse1) [reply config/routes.rb:7]: ## Hello @rukundoeric 👋 

First of all, thank you very much for your review an suggestions.

I just finished to implement your suggestions, but I have a doubt. When should I display all posts from database whithout consider users?, is it a function for *See all posts* button?. I saw at project screenshots that it display all posts for a given user, including their ocmments.

This is my last chance to request a review. Please, I would like to ask you to review my changes, before I ask the last review.

Thank you very much in advance.

**Review by arslanbisharat — APPROVED** [review]
Hi,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**